orchestra/models/communication/models.py

```diff
@@ -12,7 +12,7 @@ class CommunicationPreference(CommunicationPreferenceMixin, models.Model):
         contacted for a given CommunicationType.
 
         Attributes:
-            worker (orchestra.models.Worker):
+            worker (orchestra.models.core.Worker):
```

Comment:
since we import the models in `orchestra/models/__ini__.py` i think we can just leave as is